packages/resolve-gateway/README.md

```diff
@@ -0,0 +1,33 @@
+# `resolve-query`
```

Comment:
Implicit copy-paste from `resolve-query`, should be removed or replaced to appropriate readme.